requirements-dev1.txt

```diff
@@ -26,6 +27,7 @@ shap
 skl2onnx
 skorch
 tensorflow
+tf-keras
```

Comment:
Added this to deal with compatibility issues in the latest version of the transformers package

Reply:
Outdated and was resolved in a separate PR: https://github.com/operatorai/modelstore/pull/283